preprocessing/text_data/04_preprocessing_text_data_v3.ipynb

```diff
@@ -0,0 +1,1955 @@
+{
```

Comment:
<p>Typo:</p><blockquote>Most of the times</blockquote><p>should be "time" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>